pkg/24.0.0/24.0.0.diff

```diff
@@ -94,29 +94,20 @@
  		;;
  
  		*)
-@@ -553,14 +578,17 @@
- 					set -x
- 				fi
- 				$sh_c "DEBIAN_FRONTEND=noninteractive apt-get install -y -qq $pkgs >/dev/null"
-+				start_docker
```

Comment:
Why are we removing addition of `start_docker` - doesn't seem to match description of what the PR does.

Reply:
Ah! that was a mistake. Updated it.